WalletWasabi/BitcoinCore/P2pNode.cs

```diff
@@ -166,15 +167,15 @@ public void Disconnect()
 						{
```

Comment:
There you have the same situation. Create a local variable of TrustedP2pBehavior and use that.

Reply:
Done.